backend/db/utils.py

```diff
@@ -261,6 +261,21 @@ def is_table_up_to_date(table_name: str, skip_if_updated_within_hours: int = Non
     last_updated_key = f'last_updated_{table_name}'
     return check_if_updated(last_updated_key, skip_if_updated_within_hours)
 
+def is_refresh_active(local=False) -> bool:
```

Comment:
New function for determining if refresh is active based on timestamps. I added to the docstring an explanation of how this works and  what the 6 hour threshold is and why it was chosen.